src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="app-fullscreen">
+
+    <navbar></navbar>
+
+    <div class="dash-header">
+        <div class="dash-left">
+            <h2>Lab</h2>
+        </div>
+        <div class="dash-right">
+            <button type="button" class="btn btn-primary">New Project</button>
```

Comment:
Looks like we've got some custom classes defined in use elsewhere in the app; this should probably have the classes `button button-default` instead, to match what's on the 'reset' button on the charts page.